src/contracts/base-contract/base-contract.spec.ts

```diff
@@ -95,14 +95,22 @@ describe('BaseContract', function() {
     await ipfs.stop();
   });
 
-  it('can be created', async () => {
+  it.only('can be created', async () => {
```

Comment:
If `.only` does what I think it does, we shouldn't use it in production code, just for developing and testing a single spec. ;)

Reply:
Hahaha... Agreed... we definitely should **NOT** use it in production... :)